safenode/src/bin/faucet/cli/faucet.rs

```diff
@@ -0,0 +1,123 @@
+// Copyright 2023 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under The General Public License (GPL), version 3.
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. Please review the Licences for the specific language governing
+// permissions and limitations relating to use of the SAFE Network Software.
+
+//! The `faucet` subcommand is a testnet only command.
+//! It is used to create the genesis DBC and deposit it to the faucet local wallet.
+//! The idea is to then receive a public address from a community member, and send
+//! a small amount of tokens to them. This will allow them to send tokens to others,
+//! and start using the whole feature of token transfers on the network.
+//!
+//! The faucet will be managed by a simple web interface, probably using aws lambda + flask mvc or some such.
+
+use safenode::{
+    client::{Client, WalletClient},
+    domain::{
+        dbc_genesis::create_genesis,
+        wallet::{parse_public_address, DepositWallet, LocalWallet, Wallet},
+    },
+};
+
+use sn_dbc::Token;
+
+use clap::Parser;
+use eyre::Result;
+use std::path::PathBuf;
+
+#[derive(Parser, Debug)]
+pub enum FaucetCmds {
+    /// Create the genesis DBC and deposit it to the faucet local wallet.
+    Genesis,
+    Send {
+        /// This shall be the number of nanos to send.
+        /// Necessary if the `send_to` argument has been given.
```

Comment:
Pattern:
cargo run --bin faucet --release -- faucet send -- [amount per request] [public address hex]

Example:
cargo run --bin faucet --release -- faucet send -- 100 9b0f0e917cd7fe75cad2196c4bea7ed1873deec85692e402be287a7068b2c3f7b6795fb7fa10a141f01a5d69b8ddc0a40000000000000001

Reply:
mistake edit...

Reply:
> the CLI wil be : --send to xxxxxx or --send_to xxxxxx ?

But now I see what you said there as well, the comment needs to be updated to `to` instead of `send_to`.

Reply:
Fixed.

Reply:
better put your above example https://github.com/maidsafe/safe_network/pull/165#discussion_r1175205279
into the code comment, or a ReadMe ?

Reply:
Yes, that's better. Will fix!